Formula/e/erofs-utils.rb

```diff
@@ -24,6 +24,7 @@ class ErofsUtils < Formula
   depends_on "pkg-config" => :build
   depends_on "lz4"
   depends_on "util-linux" # for libuuid
+  depends_on "xz"
```

Comment:
macOS provides `xz` (liblzma). Can that be used?

If so, we can use

```ruby
uses_from_macos "xz"
```

Reply:
erofs-utils needs to use xz >= 5.4. I'm not sure if the native macOS liblzma version is enough.

Reply:
That's very new. Okay, let's use the `xz` formula then.